beest/keeper.dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM stackstate/stackstate-agent-2-test:beest-keeper-base-20220108
```

Comment:
Same thing about the date here. We should parameterize it

Reply:
agreed